src/components/CollaboratorModal/components/MainSubmodal.tsx

```diff
@@ -148,20 +148,10 @@ export const MainSubmodal = ({
   ...props
 }: MainSubmodalProps): JSX.Element => {
   const { siteName } = useParams<{ siteName: string }>()
+  const [errorMessage, setErrorMessage] = useState<string>("")
```

Comment:
i can't seem to find where this is used?

Reply:
Hmm the `errorMessage` is used here:

https://github.com/isomerpages/isomercms-frontend/blob/0a7c10964894e9cfa85cae8b8d9d44d01b2d0b13/src/components/CollaboratorModal/components/MainSubmodal.tsx#L233

This has always existed. The main difference is how this `errorMessage` is created. Previously it was a `useEffect` hook and now it has shifted to a callback function within the `react-query` hook.

Reply:
oh thanks! i saw that this was a new change so i was wondering where it's used (didn't review in vscode)